apps/src/code-studio/progressRedux.js

```diff
@@ -327,6 +327,7 @@ export const levelsByLesson = state => (
         progression: level.progression,
         icon: level.icon,
         isUnplugged: level.kind === LevelKind.unplugged,
+        isGoBeyond: level.title === 'Go Beyond',
```

Comment:
I think level.title is localized (i.e. if we're not in English, it won't be "Go Beyond")